test/built-ins/Temporal/PlainDateTime/prototype/withPlainDate/argument-string-iso-calendar.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.withplaindate
+description: New calendar is preserved if original PDT has ISO calendar
+features: [Temporal]
+---*/
+
+const dt = new Temporal.PlainDateTime(1995, 12, 7, 3, 24, 30);
+const shifted = dt.withPlainDate("2008-09-06[u-ca=japanese]");
```

Comment:
This will fail on non-ECMA-402 hosts. But it's not possible to create a PlainDate from a string with a custom calendar, so maybe this test should be moved to `intl402/`.